src/Widget.php

```diff
@@ -64,10 +74,14 @@ protected function getId(): ?string
      *
      * @return array
      */
-    protected function addOptions(array $options, string $valueDefault): array
+    protected function addOptions(array $options, string $valueDefault, array $params = []): array
```

Comment:
That looks a bit weird... why do we have two sources for the same `$options`?

Reply:
Fixed.